tasks.py

```diff
@@ -52,8 +52,8 @@ def uwsgi(
 
     command_args = [
         "uwsgi",
```

Comment:
uWSGI is not currently integrated into the env, maybe it's better to handle it in another PR